Opcodes/oscbnk.c

```diff
@@ -2017,7 +2017,7 @@ static void vco2_calculate_table(CSOUND *csound,
   /* inverse FFT */
   fftbuf[1] = fftbuf[table->size];
   fftbuf[table->size] = fftbuf[(int32_t) table->size + 1] = FL(0.0);
-  csound->InverseRealFFT(csound, fftbuf, (int32_t) table->size);
+  csound->RealFFT(csound,csound->RealFFTSetup(csound,table->size,FFT_INV),fftbuf);
```

Comment:
This call to RealFFTSetup looks awkward inline. Would prefer this be made into a separate variable that's initialized on a separate line.